### ISSUE TITLE
Navbar links with `.active` applied don't have a unique `:focus` state

### ISSUE THREAD

**Issue by cjg89** [UCF/Athena-Framework#232]
**Describe the bug**

See title.  Styles for `:focus` and `.active` are identical on these links, so keyboard users don't have a visual indicator when they've tabbed to one of these links.

**Steps to Reproduce**

Try tabbing to the "home" link in [the first example here.](https://ucf.github.io/Athena-Framework/components/navbar/#supported-content)

**Expected Behavior**

All navbar links should have a unique focus state, regardless of their `.active` state.

**Possible Fix**

Consider re-enabling outlines around nav links, but using `:focus-visible` instead of `:focus` in supported browsers.

**Your environment:**

n/a

**Additional context**

n/a
